BasicRxJavaSampleKotlin/app/build.gradle

```diff
@@ -21,11 +21,10 @@ apply plugin: 'kotlin-android-extensions'
 
 android {
     compileSdkVersion build_versions.compile_sdk
-    buildToolsVersion build_versions.build_tools
 
     defaultConfig {
         applicationId "com.example.android.observability"
-        minSdkVersion build_versions.min_sdk
+        minSdkVersion 21
```

Comment:
`build_versions.min_sdk` exists in the `versions.gradle` file

Reply:
min_sdk is quite low so we only use it in some samples. This one breaks with that.